ansible/roles/php/tasks/php-cli.yml

```diff
@@ -1,10 +1,10 @@
 ---
+- name: enabling opcache cli
+  lineinfile: dest=/etc/php/7.0/cli/php.ini
+              regexp=';?opcache.enable=\d'
+              line='opcache.enable=1'
```

Comment:
why do we need opcache enabled on the machine? It is no production machine after all